devices/ble_hci/common-hal/_bleio/att.c

```diff
@@ -857,20 +857,6 @@ STATIC void process_find_info_req(uint16_t conn_handle, uint16_t mtu, uint8_t dl
     }
 }
 
-int att_find_info_req(uint16_t conn_handle, uint16_t start_handle, uint16_t end_handle, uint8_t response_buffer[]) {
-    struct __packed req {
-        struct bt_att_hdr h;
-        struct bt_att_find_info_req r;
-    } req = { {
-                  .code = BT_ATT_OP_FIND_INFO_REQ,
-              }, {
-                  .start_handle = start_handle,
-                  .end_handle = end_handle,
-              }};
-
-    return send_req_wait_for_rsp(conn_handle, sizeof(req), (uint8_t *)&req, response_buffer);
-}
-
```

Comment:
Please don't delete this; the _bleio HCI implementation is incomplete and this may be needed later. Let's just comment it out for now.